crates/breez-sdk/core/src/sdk.rs

```diff
@@ -736,6 +753,25 @@ impl BreezSdk {
 
         tokio::join!(sync_wallet, sync_lnurl, sync_deposits);
 
+        if sync_type.contains(SyncType::ConversionInfo) {
```

Comment:
My concern here is that we always attempting to approach flashnet API while lots of users won't use it. Slowing the sync for most cases for no reason.
Perhaps we can add a config flag (enable_flashnet) to signal we need to sync with it?